scripts/data_transfer/copy_geneloc_genotype_files.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -ex
+
+# copy chromosome 7 and 22 geneloc files fmor main to test
+gsutil cp gs://cpg-tob-wgs-main/scrna-seq/grch38_association_files/gene_location_files/GRCh38_geneloc_chr7.tsv gs://cpg-tob-wgs-test/scrna-seq/grch38_association_files/gene_location_files/
+gsutil cp gs://cpg-tob-wgs-main/scrna-seq/grch38_association_files/gene_location_files/GRCh38_geneloc_chr22.tsv gs://cpg-tob-wgs-test/scrna-seq/grch38_association_files/gene_location_files/
+
+# copy genotype output from eQTL association pipeline into analysis to explore data
+gs://cpg-tob-wgs-main-tmp/scrna-seq/eqtl_output/v3-scale-test/genotype_table.mt gs://cpg-tob-wgs-main-tmp/scrna-seq/eqtl_output/v3-scale-test/
```

Comment:
This is copying to `tmp` instead of `analysis`, but more importantly we shouldn't copy this as a one-off, but change the pipeline to write to that destination instead.

```suggestion
```

Reply:
Sorry, that was a copy paste typo on my end. I agree we should change where this is output in the pipeline, but for now to debug, should we still copy this file from `tmp` to `analysis`? Or should we only change this in the association script and regenerate it by rerunning?

Reply:
I'd prefer regenerating it, otherwise it'll be unclear later on where this came from.